packages/migrate/src/__tests__/DbSeed.test.ts

```diff
@@ -175,21 +78,130 @@ Please provide the --preview-feature flag to use this command.
 })
 
 describe('seed - legacy', () => {
+  it('no seed file', async () => {
```

Comment:
I'm not clear on the `seed - legacy` tests. Are we keeping the old implementation around?

Reply:
Not sure how to make it clearer, it's about testing what happens in a legacy setup / deprecated things

Reply:
ah ok, re-reading this now with that in mind

Reply:
"legacy setup upgrade" would have worked